tests/CoinAddressValidationTests.cpp

```diff
@@ -237,4 +237,9 @@ TEST(Coin, validateAddressCosmos) {
     EXPECT_FALSE(validateAddress(TWCoinTypeCosmos, "xosmos1hkfq3zahaqkkzx5mjnamwjsfpq2jk7z0emlrvp"));
 }
 
+TEST(Coin, validateAddressRavencoin) {
+    EXPECT_TRUE(validateAddress(TWCoinTypeRavencoin, "RHoCwPc2FCQqwToYnSiAb3SrCET4zEHsbS"));
+    EXPECT_FALSE(validateAddress(TWCoinTypeRavencoin, "XHoCwPc2FCQqwToYnSiAb3SrCET4zEHsbS"));
```

Comment:
same here

Reply:
Done in 04d3f6d7107d0a235448acd843fb1504b1814b6d